pkg/utils/leaderelection/leaderelection.go

```diff
@@ -0,0 +1,36 @@
+package leaderelection
```

Comment:
this can probably put in `pkg/leaderelection` (skip `utils`) but not a big deal, and we can move later